client/scripts/controllers/chain/ethereum/commonwealth/protocol.ts

```diff
@@ -174,16 +177,16 @@ export default class CommonwealthProtocol {
     projectHash: string,
   ) {
     // router.post('/cw/redeem-bToken', passport.authenticate('jwt', { session: false }), backProject.bind(this, models));
-    const res = await $.post(`${app.serverUrl()}/cw/back-project`, {
-      backer,
-      amount,
-      projectHash,
-      auth: true,
-      jwt: app.user.jwt
-    });
-    if (res['status'] === 'Success') {
-      await this.updateState();
-    }
+    // const res = await $.post(`${app.serverUrl()}/cw/back-project`, {
+    //   backer,
+    //   amount,
+    //   projectHash,
+    //   auth: true,
+    //   jwt: app.user.jwt
+    // });
+    // if (res['status'] === 'Success') {
+    //   await this.updateState();
+    // }
```

Comment:
This and lots of the protocol controller post routes seem to be commented out. Why?

Reply:
since we use real chain data, we will not use local API test data now.